### PR TITLE
Update 10.blackbox.md

### PR DESCRIPTION
**Pull request by ejfertig**
Enhanced interpretability of feature selection and importance for clinical tests.

**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/10.blackbox.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ ML models vary widely in terms of interpretability: some are fully transparent
 Logistic regression and decision tree models are generally considered interpretable, while deep neural networks are often considered among the most difficult to interpret because they can have many parameters and non-linear relationships. 
 
 Model interpretability is particularly important in biomedicine, where subsequent decision making often requires human input.
-Many authors attribute a lower uptake of DL tools in healthcare to interpretability challenges [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
+For example, while prediction rules can be derived from high-throughput molecular datasets, most affordable clinical tests rely on lower dimensional measurements of a limited number of biomarkers. Many authors attribute a lower uptake of DL tools in healthcare to interpretability challenges [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x].
```
Comment (cgreene) [inline content/10.blackbox.md:10]:
```suggestion
For example, while prediction rules can be derived from high-throughput molecular datasets, most affordable clinical tests rely on lower dimensional measurements of a limited number of biomarkers.
Selecting those biomarkers to support decision making is an important modeling and interpretation challenge.
Many authors attribute a lower uptake of DL tools in healthcare to interpretability challenges [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
```

File: content/10.blackbox.md [hunk 2]
```diff
@@ -17,4 +17,4 @@ When deciding on a machine learning approach and model architecture, consider an
...
 On the other hand, if features must be combined in complex ways to be meaningful for the task, the performance difference of a model capable of capturing that structure may outweigh the interpretability costs.
-An appropriate choice can only be made after careful consideration, which often includes estimating the performance of a simple, linear model that serves as a [baseline](#baselines).
+An appropriate choice can only be made after careful consideration, which often includes estimating the performance of a simple, linear model that serves as a [baseline](#baselines). In cases where models are learned from high-throughput datasets, a small subset of features in the dataset may be strongly correlated with the complex combination of the larger feature set defined from the deep learning model. In this case, this more limited number of features can themselves be used in the subsequent simplified model to further enhance intepretability of the model.  This feature reduction can be essential to defining  biomarkers to enable clinical applications of deep learning models.
```
Comment (cgreene) [inline content/10.blackbox.md:20]:
```suggestion
An appropriate choice can only be made after careful consideration, which often includes estimating the performance of a simple, linear model that serves as a [baseline](#baselines).
In cases where models are learned from high-throughput datasets, a small subset of features in the dataset may be strongly correlated with the complex combination of the larger feature set defined from the deep learning model.
In this case, this more limited number of features can themselves be used in the subsequent simplified model to further enhance interpretability of the model.
This feature reduction can be essential to defining biomarker panels that enable clinical applications.
```

**Review by cgreene — APPROVED** [review]
I like these changes - I made them one sentence per line and suggested a change here and there, but 🚀

File: content/10.blackbox.md (hunk 1 above)
Comment (Benjamin-Lee) [reply content/10.blackbox.md:10]: Good suggestion on the transition @cgreene

**Pull request merged**